### DISCUSSION TITLE
ADD SPECTRAL TOOL AND NOISE GATE BACK

### DISCUSSION THREAD

**Discussion by MisaghMoshiri** [audacity/audacity#7418] [Polls]
in the update 3.6.4, with many great changes, i noticed that spectral tools were deleted which made me roll back to the previous update 3.6.3, i commonly use spectral delete and spectral edit multi tool to remove the sharp sounds and it is very effective in My podcasts, I couldn't find anything else doing the trick because all other tools affect all the selected area while for example i want to remove a dot or fade it to the way that it would be there but not gone just weak so it wouldn't bother the ear. so, i want to ask you to add this feature back
while we're at it 
Noise Gate Was Also Deleted, That Tool has A lot of Uses like Removing Echo Without suppressing the voice or... Because Noise removal tool doesn't have threshold sliders like the noise gate has and it's something entirely different.
i Admired Open Vino Plugin Which Does Most of The Work in noise removing, but it isn't Perfect, i won't remove the echo for example, and it changes the spectral to flat and makes it leveled which is good, but you have to do it after making changes to the voice or it would be hard to find the spots that stands out in the sound.

**Comment by LWinterberg** [comment]
These tools have not been removed. But upon upgrading they sometimes can think of themselves as plugins, and appear as a plugin developed by Paul Licameli instead. I believe going to Effects -> Plugin manager -> Rescan makes them appear in the right place again.

**Reply by MisaghMoshiri** [reply]
Strange that rescanning didn't brought it up, i have to downgrade it again
